src/site/_includes/components/Img.js

```diff
@@ -93,6 +92,21 @@ const Img = function (args) {
     imgTag = html`<a href="${fullSrc}">${imgTag}</a>`;
   }
 
+  if (caption) {
+    imgTag = html`${imgTag}
+      <figcaption class="w-figcaption">
+        ${caption === true ? alt : caption}
```

Comment:
Wouldn't caption already be truthy because of the check on line 95? Should this be `${caption}` instead?